### ISSUE TITLE
diacritics-neutralise search plugin doesn't apply to html columns

### ISSUE THREAD

**Issue by hading** [DataTables/Plugins#394]
It would be nice if this could apply, if desired, to columns detected as html as well as string, ala the accents-neutralise plugin.

If one wanted to take that more or less directly, something like:

```javascript
jQuery.fn.DataTable.ext.type.search.html = function (s) {
  return removeDiacritics(s.replace( /<.*?>/g, '' ));
};
```

**Comment by DataTables** [comment]
Agreed. At the moment you'd need a second plug-in that would strip the HTML.

More generally, it might be useful to have DataTables automatically expand each plug-in to remove the HTML (or not) - i.e. two plug-ins would actually be added.
